src/build_backend.py

```diff
@@ -14,7 +14,7 @@
 TAG = 'py3-none-any'
 
 
-def find_sources(srcpkg: bool) -> Iterable[str]:
+def find_sources(*, srcpkg: bool) -> Iterable[str]:
```

Comment:
I wonder if this works on RHEL 8 with py 3.6 -- I faintly remember having to revert these at some point. The tests will tell us!

Reply:
[seems to be happy](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18830-20230522-124210-c6502549-centos-8-stream-pybridge/log.html) :+1: